### ISSUE TITLE
abnormal synaptic physiology

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3425]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

abnormal synaptic physiology

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

any functional anomaly of the membrane junction site of a nerve cell to a target cell, such as another nerve cell, an effector cell, or a sensory receptor cell; transmission of nerve impulses may be mediated by chemical or by electrical means

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://www-sciencedirect-com.ezproxy.lib.utexas.edu/science/article/pii/S0163725818302031

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal nervous system physiology 

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
@sarahstofel For definition references would you please add the PMID for the reference. For the previous tickets I've been able to use the link you gave to find the PMID but in this case the link leads to a UT login page so I can't access the paper

**Comment by sbello** [comment]
MP:0021009
